src/main/java/com/google/gerrit/extensions/common/EditInfo.java

```diff
@@ -14,10 +14,13 @@
 
 package com.google.gerrit.extensions.common;
 
+import com.google.gson.annotations.SerializedName;
+
 import java.util.Map;
 
 public class EditInfo {
   public CommitInfo commit;
+  @SerializedName("baseRevision")
```

Comment:
Because the key is "baseRevision" , the "R" is upcase, but gson cannot process it.